### DISCUSSION TITLE
Shared tsconfig.json with typescript 5.6.3

### DISCUSSION THREAD

**Discussion by nikwotton** [vercel/turborepo#9275] [Help]
### Summary

Hello 👋 
I followed this setup process: https://turbo.build/repo/docs/guides/tools/typescript  for setting up a shared tsconfig.json module in my monorepo and it has been working great! I just updated typescript from 5.0.4 to 5.6.3 though and now I get `error TS6053: File '@repo/typescript-config/tsconfig.json' not found`. I haven't changed anything other than bumping the typescript version and eslint version. I've verified the tsconfig.json still exists both in the top level of my local `typescript-config` module and under `node_modules`. I've tried deleting `node_modules/` and re-running `npm I`. I'm not sure what's going on or what else to try - hoping someone here has ran into this before and knows how to fix it

For extra context, I'm using `npm` for packages and it's an `npm workspaces` project. The command I'm trying to run that's failing is `turbo test` which delegates to `ts-node --esm ./test/unit.ts`.

**Comment by nikwotton** [comment]
For additional context, I've narrowed down the version window a little - 5.2.2 works fine but 5.3.2 breaks

**Reply by anthonyshew** [reply]
This sounds like an issue you may want to report to the TypeScript team.
